planet/cli/subscriptions.py

```diff
@@ -32,7 +33,7 @@ def subscriptions(ctx, base_url):
 # don't want to clobber Python's built-in "list". We'll define the
 # command function as "list_subscriptions".
 @subscriptions.command(name="list")
-@click.option('--pretty', is_flag=True, help='Pretty-print output.')
+@pretty
```

Comment:
I believe you need to remove line 51 and add a `@limit` here.

Reply:
good catch, will do!